build/testdata/063.bzl.golden

```diff
@@ -7,3 +7,7 @@ foo(b = c, **e)
 foo(*d, **e)
 foo(a, a2, b = c, b2 = c2, *d, **e)
 foo(baz * 2, bar = bar(z, x = y))
+foo(
```

Comment:
I was falling back to the logic used in build mode, which sets the format to multiline mode for sequences with a size greater than 1, that do not have either ForceMultiLine or ForceCompact set.
https://github.com/bazelbuild/buildtools/blob/master/build/print.go#L743]

I will remove this behaviour for bzl/default mode.

Reply:
The BUILD-file logic only makes sense for top-level statements because nested statements are not allowed there by Bazel (although buildifier doesn't restrict that).

For nested statements and default/.bzl files different logic should apply. E.g. `a.get(b, f(x, y)` shouldn't be formatted by default as

    a.get(
        b,
        f(
            x,
            y,
        )
    )